tests/ui/process/multi-panic.rs

```diff
@@ -8,10 +8,17 @@ fn check_for_no_backtrace(test: std::process::Output) {
     let err = String::from_utf8_lossy(&test.stderr);
     let mut it = err.lines();
 
+    assert_eq!(it.next().map(|l| l.is_empty()), Some(true));
     assert_eq!(it.next().map(|l| l.starts_with("thread '<unnamed>' panicked")), Some(true));
     assert_eq!(it.next().is_some(), true);
-    assert_eq!(it.next(), Some("note: run with `RUST_BACKTRACE=1` \
-                                environment variable to display a backtrace"));
+    assert_eq!(
+        it.next(),
+        Some(
+            "note: run with `RUST_BACKTRACE=1` \
+                                environment variable to display a backtrace"
+        )
+    );
+    assert_eq!(it.next().map(|l| l.is_empty()), Some(true));
```

Comment:
I think it's better to filter out blank lines here so that the test isn't too sensitive to such changes.
The function name says it's supposed to test for the absence of backtraces, so blank lines shouldn't be relevant.

Reply:
This point is still open though.